socialhome/streams/templates/streams/base.html

```diff
@@ -5,11 +5,12 @@
 {% block containertype %}container-fluid{% endblock %}
 
 {% block ogtags %}
-    <meta property="og:title" content="{% block stream_title %}{% endblock %} - {{ request.site.name }}" />
-    <meta property="og:type" content="website" />
-    <meta property="og:url" content="https://{{ request.site.domain }}{{ request.path }}" />
+    <meta property="og:title" content="{% block stream_title %}{% endblock %} - {{ request.site.name }}"/>
```

Comment:
There are no actual changes related to this PR in this file, just reorganizing code. Can we leave it as is? Template code starts to look a bit ugly if you start breaking the lines too aggressively, and especially if it's multi-line but closing tags are not on the same level as the opening tag. In any case, since these changes are not related to the PR, it would be nicer to leave this file as it was.